src/main/target/PYRODRONEF4/target.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define TARGET_BOARD_IDENTIFIER "PYRO"
+#define USBD_PRODUCT_STRING "PYRODRONEF4"
+
+#ifdef OPBL
```

Comment:
Will there be an OPBL version of this firmware? Or why else is this in here?

Reply:
@mikeller I follow the don't know. I was looking through the code and some of them have this in it. and commented remove this at major release(?)
should I just delete it?

Reply:
Yes, please delete. OPBL (open pilot boot loader) is a boot loader that is used by some firmwares, but not Betaflight. Some boards that ship with non-Betaflight firmware come with it preloaded, and there is (or was) a Betaflight firmware version that works with OPBL, in order to make it easier for users to switch back and forth between firmwares. For all other boards, using OPBL does not make sense with Betaflight.

Reply:
@mikeller got it. will do. thanks.